src/main/java/com/mycompany/cajeroautomatico/CajeroAutomatico.java

```diff
@@ -1,187 +0,0 @@
-
-package com.mycompany.cajeroautomatico;
-
-import java.io.IOException;
-
-public class CajeroAutomatico {
-
-    public static void main(String[] args) throws IOException {
-        Consola consola = new Consola();
-        Cliente cliente = new Cliente();
-        CC cc = new CC ();
-        CAPesos ca = new CAPesos();
-        CADolares dol = new CADolares();
-        Movimiento movimiento = new Movimiento();
-        LectorDeArchivos lector = new LectorDeArchivos();
-        
-        //Bienvenida y menu principal
-        String numTarjeta = consola.bienvenida();
-        String pin =  consola.pin();
-        cliente.validacion(numTarjeta, pin);
-        for (Cuenta cuenta: cliente.getMisCuentas()){
```

Comment:
No le veo sentido a este for. Un for en un main es un peligro en cualquier app. Esto es una función. No debería estar aca, y tamopco debería ser así